CelebrityStalker.xcodeproj/project.pbxproj

```diff
@@ -34,6 +35,7 @@
 		3FFB72E61BD811C4000E3300 /* CelebrityTabBarController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = CelebrityTabBarController.swift; sourceTree = "<group>"; };
 		3FFB72E81BD9273B000E3300 /* Main.storyboard */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = file.storyboard; name = Main.storyboard; path = ../../../Downloads/Main.storyboard; sourceTree = "<group>"; };
 		3FFB72EA1BD93342000E3300 /* EditCelebrityDetailsViewController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = EditCelebrityDetailsViewController.swift; sourceTree = "<group>"; };
+		A1C9FAA11BD9369300161C68 /* celebBio.JSON */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.json; path = celebBio.JSON; sourceTree = "<group>"; };
```

Comment:
*This is simply adding the **JSON file** to the **PBXFileReference element** of the project.*